src/test/resources/com/jpinpoint/perf/lang/java/ruleset/common/xml/NonComparableMapKeys.xml

```diff
@@ -72,5 +72,22 @@ public class Issue470 {
      ]]></code>
     </test-code>
 
+    <test-code>
+        <description>no violation: prevent false positives in NonComparableMapKeys and NonComparableSetElements #568</description>
+        <expected-problems>0</expected-problems>
+        <code><![CDATA[
+import java.util.*;
+
+public class NonComparableMapKeys {
+    Map<NonComparableKey, String> nckMap; // <-- false positive, cannot be resolved, should be assumed negative
```

Comment:
why "should be assumed negative"? -> better "assume Comparable to avoid false positive"